src/platforms/android/common/performance/included-instrumentation.mdx

```diff
@@ -107,3 +107,13 @@ fun displayUserData() {
 ```
 
 When you didn't finish the transaction yet, but you start a new Activity, the SDK always automatically finishes the previous Activity transaction. This is due to the fact that only one transaction at a time can be bound to the Scope. To work around that, you may create transactions manually using the custom instrumentation and its instance to start spans instead of the Static API.
+
+### OkHttp library Instrumentation
+
+The OkHttp's instrumentation, once added the `SentryOkHttpInterceptor`, starts a span out of the active span bound to the scope for each HTTP Request. The SDK sets the span `operation` to `http.client` and `description` to request `$method and $URL`; for example, `GET https://sentry.io`.
```

Comment:
`m`: If you are new to Sentry and have no clue what a span, an active span, or a scope is, you might have difficulty understanding what this is doing. Can we maybe describe on a higher, more abstract level what this is doing before going into such detail?

Reply:
to come to this page, people already went thru https://sentry-docs-git-feat-okhttp.sentry.dev/platforms/android/guides/okhttp/performance/ which links to https://sentry-docs-git-feat-okhttp.sentry.dev/product/performance/distributed-tracing/ as well, so I guess we don't need it, it's a subpage of the performance docs